src/utilities/telegram_utils.rs

```diff
@@ -51,16 +89,22 @@ pub const fn get_message_text(content: &MessageContent) -> Option<&FormattedText
     Some(formatted_text)
 }
 
-pub fn get_message_image(content: &MessageContent) -> Option<MessageImage> {
```

Comment:
do not remove this method. it's used in other parts of code

Reply:
I replaced the other parts with my enum, which is more versatile. Are you sure you want it back?

Reply:
does this enum allow only requesting an image, and not other media?